src/background.js

```diff
@@ -3,7 +3,7 @@ const FACEBOOK_CONTAINER_NAME = "Facebook";
 const FACEBOOK_CONTAINER_COLOR = "blue";
 const FACEBOOK_CONTAINER_ICON = "briefcase";
 const FACEBOOK_DOMAINS = [
-  "facebook.com", "www.facebook.com", "fb.com", "fbcdn.net", "cdn.fbsbx.com",
+  "facebook.com", "www.facebook.com", "fb.com", "fbcdn.net", "cdn.fbsbx.com","lookaside.fbsbx.com",
```

Comment:
The plugin uses a regex match, so it would be more-correct to have one `fbsbx.com` record, and remove any subdomain references

Reply:
yeah, any duplication that's going on could, and should be simplified. that way we'll hopefully get @groovecoder to merge this into the repo.

Reply:
@nemoload can you address @NewAlexandria 's comment in your PR?

Reply:
Sure, no problem.